packages/dmn-js-decision-table/assets/css/dmn-js-decision-table-controls.css

```diff
@@ -285,7 +285,8 @@
   box-shadow: 1px 1px 1px 1px var(--edit-button-box-shadow-color);
 }
 
-.dmn-decision-table-container :not(:focus-within) .edit-button {
+.dmn-decision-table-container :not(:focus-within) .edit-button,
+.dmn-decision-table-container :has(.dmn-icon-plus:active) .edit-button {
```

Comment:
Can this solution be [signed-off from the maintainer perspective](https://github.com/bpmn-io/internal-docs/blob/main/development/DEFINITION_OF_DONE.md)? What if we replace the plus icon with something else?

Reply:
Should the edit button visibility be driven by another component?

Reply:
Thanks, refactored.